pyclaragenomics/setup.py

```diff
@@ -11,6 +11,7 @@
 #
 
 import os
+import glob
```

Comment:
Imports should be ordered with stdlib first and 3rd party after, alphabetical within groups like so:

```
import glob
import os
from setuptools import setup, find_packages, Extension

from Cython.Build import cythonize
```